code-challenges.test.js

```diff
@@ -10,8 +10,22 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-// --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
 
+
+
+
+
+
+
+
+
+
+// --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
+describe("TheCapper", () => {
+  it("returns an array with a sentence about each person with their name capitalized.", () => {
+    expect(TheCapper(people)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."])
```

Comment:
The test is set up nicely just make sure you are using the right naming convention for functions.